docs/getstarted/core_concepts.md

```diff
@@ -0,0 +1,146 @@
+# Core Concepts
+
+FloydHub is a platform that enables data scientists use to create deep learning
+models more efficiently. FloyHub removes the of the hassle of maintaining and
+provisioning deep-learning servers, allowing data scientists to focus on the
+science of deep learning. It's deep learning without the DevOps.
```

Comment:
The Home section is already dedicated to platform introduction. Do we still need to add the introduction in this section?

Reply:
Yeah you make a good point. Let me think of a better intro here.